pkg/injector/test/.gitignore

```diff
@@ -0,0 +1 @@
+actual_*.yaml
```

Comment:
if you use `mktemp` and compare to these, we won't have to these files as part of the code base (and thus have to ignore them).

Reply:
Great idea - will use `ioutil.TempDir("", "envoy")`